### DISCUSSION TITLE
Display Time Axis (tick-marks and labels) as UNIX Timestamps

### DISCUSSION THREAD

**Discussion by jonnyzwi** [tradingview/lightweight-charts#804] [Q&A] [answered]
I realize this is the opposite of what people are usually hoping to achieve, but I would like to display ticks and labels on the time axis as a **_UNIX timestamp_**. 

[From this post](https://stackoverflow.com/questions/65785045/how-to-set-a-custom-time-zone-for-lightweight-charts), I am aware that one can override `tickMarkFormatter`, `dateFormat` and `timeFormatter`, for the sake of formatting dates. However, I am still unsure as to how I can display time/dates as UNIX timestamps.

Thanks for any help.

**Comment by timocov** [comment, marked as answer]
> However, I am still unsure as to how I can display time/dates as UNIX timestamps.

What do you mean? If you will override these properties you can return just UNIX timestamps value as a string instead of formatted date.

**Reply by jonnyzwi** [reply]
Thanks timocov, but I have been unable to successfully do what you just described (i.e., override the other properties in order to return a UNIX timestamp).  

Could you please provide a quick example.  Thank you.

**Reply by timocov** [reply]
Try something like that https://jsfiddle.net/rbx03jz4/ (it is rough example, but just to show the concept)

**Reply by jonnyzwi** [reply]
Thank you !!
